src/WPF/WPF.Viewer/Samples/Analysis/QueryFeatureCountAndExtent/QueryFeatureCountAndExtent.xaml.cs

```diff
@@ -51,6 +66,9 @@ private async Task Initialize()
             // Add the feature layer to the map.
             myMap.OperationalLayers.Add(myFeatureLayer);
 
+            // Populate the ComboBox with states.
+            foreach (var state in _states) { StatesComboBox.Items.Add(state.Key); }
```

Comment:
I think `foreach` loops look better over multiple lines. We should include this in our coding standards.